### DISCUSSION TITLE
[Bug Report][3.2.1] testing-library can't interact with vuetify components

### DISCUSSION THREAD

**Discussion by lucaspagliari** [vuetifyjs/vuetify#17296] [Q&A] [answered]
### Environment
**Vuetify Version:** 3.2.1
**Vue Version:** 3.2.47
**Browsers:** Chrome 112.0.0.0
**OS:** Windows 10

### Steps to reproduce
run unit tests:
npm run test:unit
check files: TestC.vue and TestC.spec.ts 

### Expected Behavior
userEvent or fireEvent to really type in inputs

### Actual Behavior
events does not cause any change on vuetify components

![image](https://user-images.githubusercontent.com/42659693/235284930-26b95541-5da1-4cb1-958c-4de665769187.png)

### Reproduction Link
[https://github.com/lucaspagliari/vuetify-testing-library](https://github.com/lucaspagliari/vuetify-testing-library)

### Other comments
This is my first time using vuetify with testing-library and vitest, if I missed something please let me know
My goal is to interact with any field, button, etc... and validate these changes on my template
disable a btn or show a specific text.
I've tried many configs, and i can't get this to work, if you guys have any solution I will be very thankful...

**Comment by nekosaur** [comment, marked as answer]
When you do `screen.getByTestId('field')` you're only getting the root div of the v-text-field component. You either need to further drill down and find the `<input>`, maybe with `.querySelector('input')`. Or use the `getByLabelText` function, as long as you have an id attribute on the v-text-field
